internal/app/tfsec/adapter/aws/neptune/adapt.go

```diff
@@ -2,9 +2,45 @@ package neptune
 
 import (
 	"github.com/aquasecurity/defsec/provider/aws/neptune"
+	"github.com/aquasecurity/defsec/types"
 	"github.com/aquasecurity/tfsec/internal/app/tfsec/block"
 )
 
 func Adapt(modules []block.Module) neptune.Neptune {
-	return neptune.Neptune{}
+	return neptune.Neptune{
+		Clusters: adaptClusters(modules),
+	}
+}
+
+func adaptClusters(modules []block.Module) []neptune.Cluster {
+	var clusters []neptune.Cluster
+	for _, module := range modules {
+		for _, resource := range module.GetResourcesByType("aws_neptune_cluster") {
+			clusters = append(clusters, adaptCluster(resource))
+		}
+	}
+	return clusters
+}
+
+func adaptCluster(resource block.Block) neptune.Cluster {
+	enableLogExportsAttr := resource.GetAttribute("enable_cloudwatch_logs_exports")
+	auditVal := types.Bool(false, *resource.GetMetadata())
```

Comment:
This should be a default if its not set?
```suggestion
	auditVal := types.BoolDefault(false, *resource.GetMetadata())
```